pkg/shp/initialize/init.go

```diff
@@ -0,0 +1,36 @@
+package initialize
```

Comment:
Let's remove `initialize` - this isn't something we are prepared to support in the near term.

Reply:
I have gotten rid of it in another branch already :) will also delete here, then.